src/AppBundle/Controller/ContentController.php

```diff
@@ -24,18 +24,14 @@
 use Pimcore\Model\DataObject\Manufacturer;
 use Pimcore\Model\DataObject\Service;
 use Pimcore\Translation\Translator;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 class ContentController extends BaseController
 {
-    /**
-     * @Template
-     */
     public function defaultAction()
     {
-        return [];
+        return $this->render('content/default.html.twig');
```

Comment:
default actions rely on @Template so that it is possible to template from Admin.

Reply:
Oh, good to know! I reverted that part.